tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/DumpCounterexamplesModelCheckerListener.scala

```diff
@@ -2,50 +2,46 @@ package at.forsyte.apalache.tla.bmcmt
 
 import at.forsyte.apalache.io.lir.CounterexampleWriter
 import at.forsyte.apalache.tla.bmcmt.trex.DecodedExecution
-import at.forsyte.apalache.tla.lir.{TlaEx, TlaModule}
+import at.forsyte.apalache.tla.lir.TypedPredefs.BuilderExAsTyped
+import at.forsyte.apalache.tla.lir.convenience.tla
+import at.forsyte.apalache.tla.lir.{BoolT1, TlaEx, TlaModule}
 import com.typesafe.scalalogging.LazyLogging
 
 /**
  * Observer to [[SeqModelChecker]] that dumps counterexamples to files.
  */
 object DumpCounterexamplesModelCheckerListener extends ModelCheckerListener with LazyLogging {
 
-  /**
-   * Call when [[SeqModelChecker]] encounters a counterexample. Dumps counterexample to files.
-   *
-   * The counterexample trace is written to the following files:
-   *   - `counterexample${errorIndex}.{tla,json,.itf.json}` contains the current counterexample
-   *   - `counterexample.{tla,json,.itf.json}` contains the latest counterexample
-   *
-   * @param rootModule
-   *   The checked TLA+ module.
-   * @param trace
-   *   The counterexample trace.
-   * @param invViolated
-   *   The invariant violation to record in the counterexample. Pass
-   *   - for invariant violations: the negated invariant,
-   *   - for deadlocks: `ValEx(TlaBool(true))`,
-   *   - for trace invariants: the applied, negated trace invariant (see [[SeqModelChecker.applyTraceInv]]).
-   * @param errorIndex
-   *   Number of found error (likely [[SearchState.nFoundErrors]]).
-   */
```

Comment:
This has been merged, but is there any reason why this whole doccomment was removed? I spent quite some time figuring out the interface for this.

Reply:
Does not it simply repeat the comment from `ModelCheckerListener`? If not, sorry about that!